test.sh

```diff
@@ -7,4 +7,6 @@ IMG="$REGISTRY/$REPOSITORY:$TAG"
 echo "Running bats tests"
 docker run -i --entrypoint "bats" "$IMG" "/tmp/test"
 
+./test-auth.sh ${TAG} ${IMG}
```

Comment:
- Quotes
- Consider reverting `TAG` and `IMG` here (more specific last, least specific first)